packages/react-core/src/components/Label/Label.tsx

```diff
@@ -183,17 +183,19 @@ export const Label: React.FunctionComponent<LabelProps> = ({
   const LabelComponent = (isOverflowLabel ? 'button' : 'span') as any;
 
   const button = closeBtn ? (
-    closeBtn
+    <span className={css(styles.labelActions)}>{closeBtn}</span>
   ) : (
-    <Button
-      type="button"
-      variant="plain"
-      onClick={onClose}
-      aria-label={closeBtnAriaLabel || `Close ${children}`}
-      {...closeBtnProps}
-    >
-      <TimesIcon />
-    </Button>
+    <span className={css(styles.labelActions)}>
+      <Button
+        type="button"
+        variant="plain"
+        onClick={onClose}
+        aria-label={closeBtnAriaLabel || `Close ${children}`}
+        {...closeBtnProps}
+      >
+        <TimesIcon />
+      </Button>
+    </span>
   );
```

Comment:
Super small non blocking nit: to get rid of the duplicate span I would wrap the whole button in the span, i.e. something like


```js
const defaultButton = (
    <Button
        type="button"
        variant="plain"
        onClick={onClose}
        aria-label={closeBtnAriaLabel || `Close ${children}`}
        {...closeBtnProps}
      >
        <TimesIcon />
    </Button>
)

const button = (
    <span className={css(styles.labelActions)}>
        {closeBtn || defaultButton}
    </span>
)